pennylane/templates/layers/random.py

```diff
@@ -230,6 +232,13 @@ def compute_decomposition(
          CNOT(wires=['b', 'a']),
          RX(tensor(1.4000), wires=['a'])]
         """
+        if ratio_imprimitive:
+            warnings.warn(
+                "In RandomLayers.compute_decomposition, ratio_imprim has been changed to `ratio_imprimitive` to match the "
```

Comment:
```suggestion
                "In RandomLayers.compute_decomposition, ratio_imprim should be changed to `ratio_imprimitive` to match the "
```
Just to make it an explicit request to the user.